src/SDKs/_metadata/ApplicationInsights_data-plane.txt

```diff
@@ -1,11 +1,11 @@
-2018-02-27 20:41:17 UTC
+2018-09-23 01:55:41 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: Azure
-Branch:      master
-Commit:      d122af3086f0b1693b9c506befc497a1f4a8b8e1
+GitHub user: alexeldeib
```

Comment:
Is the spec merged?

Reply:
There's an open PR which changes an enum to modelAsString. Rest of the changes are merged.

Reply:
The same open PR contains the autorest config to change namespace to Microsoft.Azure.ApplicationInsights.Query

Reply:
I can regenerate after it's merged. The same PR contains the namespace change.